mirgecom/euler.py

```diff
@@ -34,6 +34,12 @@
 
 .. autofunction:: get_inviscid_timestep
 .. autofunction:: get_inviscid_cfl
+
+References
+----------
+.. [DGBook] Hesthaven and Warburton (2008), Nodal DG Methods, Springer
+   (DOI: 10.1007/978-0-387-72067-8)
```

Comment:
Put the references all in one place. (Rather than scattering them around the docs.)